### DISCUSSION TITLE
Postman don't save durables after 2nd End of Loop block

### DISCUSSION THREAD

**Discussion by lHuKKal** [postmanlabs/postman-flows#367] [Defect]
Hello!

I have a problem. In my flows have 2 the End Loop and Create Durables blocks. When i starting my flow and the flow reaches the 1st End Loop block, the postman saved 1st durables (IDUser). After as the flow reaches the 2nd End Loop block, the postman saved 2nd durables (IDReason), BUT 1st durables (IDUser) is not displayed in terminal 

Example: 
Here, the postman saved durable (IDUser) after 1st End Loop block and ID is displayed in the terminal:
![1st](https://user-images.githubusercontent.com/97214688/183824797-baa4c320-89b8-43f1-b4ce-3a732c2d1bb5.png)

Here. the postman saved 2nd durable(IDReason), BUT 1st durable (IDUser) is missing and not displayed in the terminal:
![2nd](https://user-images.githubusercontent.com/97214688/183824949-d5edee80-fadb-482f-959f-31f27e20e279.png)

But 1st durable was saved before as the flow reaches the 2nd End Loop block
![Before](https://user-images.githubusercontent.com/97214688/183825088-cf5da633-5e75-46c4-9292-12902716b777.png)

Can you check pls! It's the bug? As possible to save 2 durables with these blocks? I need to 2 durables with these blocks and with logic flows as in the example for further testing. Help pls =)

**Comment by saswatds** [comment]
Hi @lHuKKal, thanks for trying out flows. Both the durables should have been persisted, it indeed looks like a bug. Will get it fixed ASAP.

**Reply by lHuKKal** [reply]
Thanks for your reply. Let me know when u fix it, pls

**Reply by saswatds** [reply]
I have fixed the bug internally. Should be resolved in the next release.

**Reply by lHuKKal** [reply]
Hello! I have tested after updating the postman and the defect is not fixed (Version 9.29.3)

**Reply by saswatds** [reply]
Hey @lHuKKal. I just checked with the team, the fix is going out on release v9.30.0. Thanks a lot for your patience.
